src/openrct2/audio/audio.h

```diff
@@ -55,20 +55,21 @@ struct rct_ride_music_params
     uint16_t frequency;
 };
 
+struct Sound
+{
+    SoundId Id;
+    int16_t Volume;
+    int16_t Pan;
+    uint16_t Frequency;
+    void* Channel;
+};
+
 struct rct_vehicle_sound
 {
     uint16_t id;
     int16_t volume;
-    SoundId sound1_id;
-    int16_t sound1_volume;
-    int16_t sound1_pan;
-    uint16_t sound1_freq;
-    SoundId sound2_id;
-    int16_t sound2_volume;
-    int16_t sound2_pan;
-    uint16_t sound2_freq;
-    void* sound1_channel;
-    void* sound2_channel;
+    Sound Sound1;
```

Comment:
Make this into an array rather than have a sound 1 and 2

Reply:
I don't think this should be done.
Rather we should name the two sounds properly. Sound1 is TrackSounds and Sound2 is OtherSounds.

Reply:
Okay that would also work. Go for that. I was thinking we could use more ranged based for loops if they were an array